test/guest/fedora-atomic.setup

```diff
@@ -18,12 +18,17 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
+# Use mac's in name policy, prevents interference from network-scripts
```

Comment:
> However, since this means we also need to figure out the metrics issue, it's probably better to do all this in another PR.

The issue is that, *drumroll*, we don't have any internal metrics for individual network interfaces. We only have metrics for the total traffic.

Thus without PCP, we don't get any data for the graphs etc on the networking pages.  The tests are not normally affected by this, but my proposed change above means that the interface is already past the "Configuring IP" stage when we call wait_for_iface, and wait_for_iface thus waits for some traffic rate to appear, which doesn't.